pkg/auth/authz_handler_test.go

```diff
@@ -163,7 +163,7 @@ func TestAuthz(t *testing.T) {
 	if err != nil {
 		panic(err)
 	}
-	srvAddr := "localhost:8082"
+	srvAddr := "localhost:8083"
```

Comment:
Maybe the port should be generated? i think that if someone will add another test it will probably copy it so at least write a comment that the port should be uniqe between tests that run a service and please add this comment to `logcontext_test.go` as well.